Tensile/Common.py

```diff
@@ -799,6 +811,43 @@
     "NumIndicesC":              2,
     "UseInitialStrides":        False,
 
+    # For OperationType == Convolution*
+    # *HW and *YX   create solution with 2 spatial dimensions.
+    # *DHW and *ZYX create solution with 3 spatial dimensions.
+    "TensorAFormat":           "NCHW",  # NCHW, NHWC, CNHW
```

Comment:
What about putting these into another data structure?  I can see it being confusing if we have both the conv- and gemm-specific options here.

Reply:
yes, see latest code.  Looks nice in the file and has cleaner separation.